src/clarity-angular/wizard/providers/button-hub.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component} from "@angular/core";
+import {Wizard} from "../wizard";
+import {TestContext} from "../../utils/testing/helpers.spec";
+import {ButtonHubService} from "./button-hub";
+import {WizardNavigationService} from "./wizard-navigation";
+import {PageCollectionService} from "./page-collection";
+
+export default function(): void {
+
+    describe("Button Hub Service", function() {
+
+        let context: TestContext<Wizard, ButtonHubTest>;
+        let buttonHubService: ButtonHubService;
+        let wizardNavigationService: WizardNavigationService;
+        let pageCollectionService: PageCollectionService;
+
+        beforeEach(function() {
+            context = this.create(Wizard, ButtonHubTest);
+            buttonHubService = context.getClarityProvider(ButtonHubService);
```

Comment:
Yes, I agree. It was a lot of code at the backend of this component. Much of it, I need to go back over. I have an action item to do that before or after the wizard is finalized.